torch/sparse/__init__.py

```diff
@@ -78,12 +78,6 @@ def __rsub__(self, other):
     def __matmul__(self, other):
         raise NotImplementedError
 
-    def __pow__(self, other):
```

Comment:
I think I misunderstood how our codegen works. When `__pow__` was defined as NotImplemented, my tests failed; when I removed it started passing, so I assumed the parent class was magically defining __pow__. I guess I should look at this more.